### PR TITLE
feat: Seaport method diagram

### PR DESCRIPTION
**Pull request by devtooligan**
In discord, @asnared suggested we include this diagram in our docs.

After adding the final "advanced" section, I reread the prior section and felt it could be simplified.  Especially because now we get into the "advanced" technique at the end with the "Seaport method".  I thought it might be more beginner friendly if we simpliefied the example. Instead of using offset to force the important bytes from the string into 0x40, I thought it was simpler if we just use a right padded value and make 3 MSTORES, 0x00, 0x20, and 0x40.  

Anyways, it's just a suggestion, I thought it was great before so I'm happy to undo these additional changes if need be.

### CODE REVIEW

File: src/tutorial/hello-world/README.md [hunk 1]
```diff
@@ -6,73 +6,65 @@ Note: This section is a continuation of ['The Basics'](/tutorial/the-basics/). I
...
     0x0d                                // [0x0d]
     0x20                                // [0x20, 0x0d]
     mstore                              // []
-    
-    // store dynamic offset at 0x00 (string encoding starts at 0x20)
-    0x20                                // [0x20]
-    0x00                                // [0x00, 0x20]
+
+    // store bytes for "Hello, world!" at 0x40
+    0x48656c6c6f2c20776f726c642100000000000000000000000000000000000000  // <- we have to right pad this because its bytes
```
Comment (abigger87) [inline src/tutorial/hello-world/README.md:39]: We can use the `__RIGHTPAD()` builtin

**Review by abigger87 — CHANGES_REQUESTED** [review]
Small nit

File: src/tutorial/hello-world/README.md (hunk 1 above)
Comment (devtooligan) [reply src/tutorial/hello-world/README.md]: fixed https://github.com/huff-language/huff-docs/pull/39/commits/8c2e792cddd3320189af72ee885283118413a8fc

**Comment by devtooligan** [comment]
@abigger87 fixed, lemme know if there's anything else i can help w on this one

**Review by abigger87 — APPROVED** [review]
LGTM

**Pull request merged**